### PR TITLE
Search default page design updates (AIC-512)

### PR DESCRIPTION
**Pull request by caguilar187**

### CODE REVIEW

File: search/src/main/java/edu/artic/search/SearchBaseFragment.kt [hunk 1]
```diff
@@ -40,7 +38,19 @@ abstract class SearchBaseFragment<TViewModel : SearchBaseViewModel> : BaseViewMo
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
-        val lm = GridLayoutManager(view.context, 5)
+        var width = resources.displayMetrics.widthPixels
+        val ten = TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP, 10f, resources.displayMetrics).toInt()
```
Comment (Cliabhach) [inline search/src/main/java/edu/artic/search/SearchBaseFragment.kt]: There's now an extension method to simplify the conversion process. Please use `edu.artic.base.utils.dpToPixels` for this and the other DIP values.

File: search/src/main/res/layout/search_app_bar_layout.xml [hunk 2]
```diff
@@ -21,17 +26,16 @@
...
         android:lines="1"
         android:maxLines="1"
-        android:padding="@dimen/marginStandard"
         android:textColor="@color/white"
         android:textColorHint="@color/hintColor"
         android:imeOptions="actionDone"
         android:inputType="text"
-
+        android:textSize="17sp"
+        android:fontFamily="@font/ideal_sans_medium"
```
Comment (Cliabhach) [inline search/src/main/res/layout/search_app_bar_layout.xml]: Please add an `android:fontFeatureSettings="tnum"` too, so numbers show up in the approved way.

File: search/src/main/java/edu/artic/search/CircularViewItemDecoration.kt [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        adapter.getItemOrNull(position)?.let {
+            if (it is SearchTextHeaderViewModel) {
+                it.text.subscribe {
+                    if (it == R.string.on_the_map) {
+                        onTheMapPosition = position
+                    }
+                }.disposedBy(disposeBag)
+            }
+            if (onTheMapPosition > 0) {
+                if (it is SearchCircularCellViewModel || it is SearchAmenitiesCellViewModel) {
```
Comment (Cliabhach) [inline search/src/main/java/edu/artic/search/CircularViewItemDecoration.kt]: Move this `if` up a line, turn into an `else if` with the `SearchTextHeaderViewModel` block.

**Comment by caguilar187** [comment]
pending approval from kerry and @Cliabhach prior to merge

**Pull request merged**